research-software-directory/RSD-as-a-service
# Fix REUSE compatibility in documentation

Fixes #373

I chose to use a `.license` file for `documentation/docs/README.md`. Excluding the whole docs directory using a `dep5` file, as preferred by @ctwhome, was unfortunately not feasible, as the reuse tool would have added license headers again if someone used it on the `README.md`. Now, the reuse tool will add copyright information to the `.license` file. I think this solution should be fine, since we only need one `.license` file here. The license headers in the other Markdown files do not interfere when building the docs.

Changes proposed in this pull request:

* removes an unused license file
* adds a `README.md.license`
* adds license headers to some other files in `documentation/`

How to test:

* Check whether documentation still renders as intended:
  ```
  cd documentation
  yarn install
  yarn dev
  ```
* open the docs in the browser: http://localhost:3030/
* verify that the following pages do not show any reuse headers / work correctly:
  * landing page
  * "Embed an IFrame"
 

PR Checklist:
